.validaterc

```diff
@@ -3,13 +3,28 @@
     "operations": {
       "no_operation_id": "off"
     },
+    "parameters": {
+      "param_name_case_convention": "off",
+      "accept_type_parameter": "off"
+    },
+    "paths": {
+      "paths_case_convention": "off"
+    },
     "schemas": {
+      "invalid_type_format_pair": "off",
       "no_schema_description": "off",
       "no_property_description": "off",
-      "inconsistent_property_type": "off"
+      "description_mentions_json": "off",
+      "inconsistent_property_type": "off",
+      "enum_case_convention": "off"
     },
     "walker": {
       "no_empty_descriptions": "off"
     }
+  },
+  "oas3": {
+    "operations": {
+      "no_request_body_name": "off"
+    }
   }
 }
```

Comment:
It makes sense to run the validator on CircleCI as a test job. However, when I run the validator locally it brings a lot of errors

Reply:
It does, but at the moment we are using an OAS 3.1 feature to define fields as possibly nullable. OAS is currently at 3.0.3 so most errors will disappear when 3.1 is released. I tried turning off all possible rules in `.validaterc` but there are still errors because of this.